src/style.css

```diff
@@ -0,0 +1,101 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;0,500;0,700;1,300;1,400&display=swap");
+
+* {
+  margin: 0;
```

Comment:
define it for the list of tags you want to reset the margin (create _reset.css)

Reply:
fix it https://github.com/mate-academy/layout_product-cards/pull/2342/files#r1050498106